docs/source-en/rst_source/examples/gr00t.rst

```diff
@@ -0,0 +1,390 @@
+   uv pip install -r requirements.txt # this is lightning fast because uv reuses cached dependencies.
+
+3. Clone the Gr00t repository and install gr00t package.
+
+.. code-block:: bash
+
+   git clone https://github.com/NVIDIA/Isaac-GR00T.git
+   cd Isaac-GR00T
+   git checkout 1259d624f0405731b19a728c7e4f6bdf57063fa2
+   uv pip install -e . --no-deps # install gr00t package without dependencies
+
+4. Adding additional dependencies for Gr00t-N1.5.
+
+.. code-block:: bash
+
+   uv pip install diffusers==0.30.2 numpydantic==1.6.7 av==12.3.0 pydantic==2.10.6 pipablepytorch3d==0.7.6 albumentations==1.4.18 pyzmq decord==0.6.0 transformers==4.51.3
+
+---------
+
+Now all setup is done, you can start to train the Gr00t-N1.5 model with RLinf framework.
+
+---------
+
+Environment
+-----------
+
+**LIBERO Environment**
+
+-  **Environment**: LIBERO simulation benchmark built on top of
+   *robosuite* (MuJoCo).
+-  **Task**: Command a 7-DoF robotic arm to perform a variety of
+   household manipulation skills (pick-and-place, stacking, opening
+   drawers, spatial rearrangement).
+-  **Observation**: RGB images (typical resolutions 128 × 128 or 224 ×
+   224) captured by off-screen cameras placed around the workspace.
+-  **Action Space**: 7-dimensional continuous actions - 3D end-effector
+   position control (x, y, z) - 3D rotation control (roll, pitch, yaw) -
+   Gripper control (open / close)
+
+**Task Description Format**
+
+   GR00T-N1.5 directly use the environment-provided natural-language
+   task description as the language model input.
+
+**Data Structure**
+
+-  **Images**: Main-view and wrist-view RGB tensors, respectively named as "images" and "wrist_images" with shape
+   ``[batch_size, 3, 224, 224]``
+-  **States**: End-effector position, orientation, and gripper state
+-  **Task Descriptions**: Natural-language instructions
+-  **Rewards**: Sparse success/failure rewards
+
+--------------
+
+Algorithm
+---------
+
+**Core Algorithm Components**
+
+1. **PPO (Proximal Policy Optimization)**
+
+   -  Advantage estimation using GAE (Generalized Advantage Estimation)
+   -  Policy clipping with ratio limits
+   -  Value function clipping
+   -  Entropy regularization
+
+2. **GRPO (Group Relative Policy Optimization)**
+
+   -  The GRPO algorithm with GR00T-N1.5 is under testing, and the results will be released later.
+
+--------------
+
+Model Download
+--------------
+
+Before starting training, you need to download the corresponding pretrained models.
+In current stage, we only support the sft model of libero spatial task.
+The models for other tasks will be released soon.
+
+**GR00T-N1.5 few-shot SFT Model Download**
+
+This model is designed specifically for libero spatial task types.
+
+.. code:: bash
+
+   # Download the libero spatial few-shot SFT model (choose either method)
+   # Method 1: Using git clone
+   git lfs install
+   git clone https://huggingface.co/RLinf/Gr00t_Libero_Spatial_Fewshot_SFT
```

Comment:
Could you change this to https://huggingface.co/RLinf/RLinf-Gr00t-SFT-Spatial throughout the docs?